src/pages/quiz-instructions-page.tsx

```diff
@@ -39,7 +40,7 @@ const QuizInstructionPage: React.FC = (): ReactElement => {
   return (
     <FlexContainer>
       <Container text>
-        <Header as="h2">{`${LEVEL_LABEL} ${armyUnitsQuizType === 'battalion' ? 1 : 2}`}</Header>
+        <Header as="h2">{`${LEVEL_LABEL} ${armyUnitsQuizType === BATTALION ? 1 : 2}`}</Header>
```

Comment:
Nice, move all the text to the a file, it is a good coding style when you want to support different locale